### PR TITLE
BXMSDOC-2743-master:  PR for Using standalone perspectives epic

### PR DESCRIPTION
**Pull request by gadeyredhat**
Hi,

Please review the Standalone perspectives story/epic.

[Documentation jira link](https://issues.jboss.org/browse/BXMSDOC-3017) 
[Rendered output](http://file.pnq.redhat.com/~gadey/BXMSDOC-2743/) 

Regards,
Gaurav Dey

### CODE REVIEW

File: docs/product-assembly_using-standalone-perspectives/src/main/asciidoc/main.adoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+// Context attribute is assembly specific and enables module reuse between assemblies.
+:imagesdir: topics/product-shared-docs/images
+:context: using-standalone-perspectives
+include::product-shared-docs/author-group.adoc[]
+
+// Purpose statement for the assembly
+As a developer, you can use the standalone perspectives from {CENTRAL} in your application.
+
+.Prerequisite
+* Ensure {CENTRAL} is deployed and is running in a web/application server.
```
Comment (michelehaglund) [inline docs/product-assembly_using-standalone-perspectives/src/main/asciidoc/main.adoc]: Should this be "Ensure {CENTRAL} is deployed and is running *on* a web/application server"

File: docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-content-manager-proc.adoc [hunk 2]
```diff
@@ -0,0 +1,12 @@
+[id='using-standalone-perspectives-content-manager-proc']
+= Using standalone content manager perspective
+
+By using the standalone content manager perspective in your application, you can create and edit your application's content and its navigation menus.
+
+.Procedure
+. Login to {CENTRAL}.
```
Comment (michelehaglund) [inline docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-content-manager-proc.adoc]: Should be: "Log in to .."

File: docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-content-manager-proc.adoc [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+. In a web browser, enter the following link in its address bar:
```
Comment (michelehaglund) [inline docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-content-manager-proc.adoc]: Might be better to reword as follows: 
Go to http://localhost:8080/business-central/kie-wb.jsp?standalone=true&perspective=ContentManagerPerspective

File: docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-content-manager-proc.adoc [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
++
+`http://localhost:8080/business-central/kie-wb.jsp?standalone=true&perspective=ContentManagerPerspective`
++
+This will open the standalone content manager perspective in the browser.
```
Comment (michelehaglund) [inline docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-content-manager-proc.adoc]: The standalone content manager perspective opens in the browser
 Instead of using a passive phrase.

File: docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-intro-con.adoc [hunk 5]
```diff
@@ -0,0 +1,8 @@
+[id='using-standalone-perspectives-intro-con']
+= Using standalone perspectives
+
+{CENTRAL} provides different specialized editors for authoring assets which have different formats. As per an asset's format a specific designer is used for editing it. {CENTRAL} has a feature that allows you to use these editors individually. This feature is known as the standalone perspective mode of the editor or simply the _standalone perspectives_.
```
Comment (michelehaglund) [inline docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-intro-con.adoc]: Per the IBM Style Guide we shouldn't use "as per". Instead use "According to.." 
However, you could simplify the 1st 2 sentences as follows:
{CENTRAL} provides specialized editors for authoring assets based on the asset's format .

File: docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-intro-con.adoc [hunk 6]
```diff
@@ -0,0 +1,8 @@
...
+
+As a developer, this provides you with the ability to embed a standalone perspective in your web application and then use it to edit rules, processes, decision tables, and so on. After embedding a perspective you can edit an asset in your own application without switching to {CENTRAL}. You can use this feature to customize your web application. In addition to standalone perspectives you can also embed standalone custom pages (dashboards) in your applications.
+
+You can access a standalone perspective by using a specific web-link in a browser with the `standalone` and `perspective` parameters. A standalone perspective's link may contain additional parameters as well.
```
Comment (michelehaglund) [inline docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-intro-con.adoc]: You can access a standalone perspective by using a specific web address.. in a browser with the `standalone` and `perspective` parameters. A standalone perspective's web address may also contain additional parameters.

Just a thought. Up to you.

File: docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-library-proc.adoc [hunk 7]
```diff
@@ -0,0 +1,20 @@
+[id='using-standalone-perspectives-library-proc']
+= Using standalone library perspective
+
+You can use the library perspective of {CENTRAL} to select a project you want to edit. You can also perform all the authoring functions on the selected project. The home page of the standalone library perspective is the *Projects* tab of the *MySpace* page.
+
+Further the standalone library perspective can be used in two ways, _with_ and _without_ using the `header=UberfireBreadcrumbsContainer` parameter. The difference between the two ways is that the link with the *_header_* parameter will display a breadcrumbs link on top of the library perspective. Using this link you can create additional *Spaces* for your projects.
```
Comment (michelehaglund) [inline docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-library-proc.adoc]: I would remove "Further" from the 1st sentence of the 2nd paragraph. Just say "The standalone library.."

Also, I would change: "The difference between the two ways is that the link with the.." to 
The difference is that the link with the..

File: docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-library-proc.adoc [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+.Procedure
+. Login to {CENTRAL}.
+. In a web browser, enter the appropriate link:
```
Comment (michelehaglund) [inline docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-library-proc.adoc]: instead of link, consider web address. Also, I looked up "URL" in the style guide and it says to avoid it. 

Basically, it says "either use "web address" or do not use a term at all. For example, write "go to".. `http://localhost:8080/business-central/kie-wb.jsp?standalone=true&perspective=LibraryPerspective&header=UberfireBreadcrumbsContainer

File: docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-standalone-custom-pages-proc.adoc [hunk 9]
```diff
@@ -0,0 +1,12 @@
+[id='using-standalone-perspectives-standalone-custom-pages-proc']
+= Using standalone custom pages(dashboards)
+
+Apart from standalone perspectives, you can also embed custom pages (also known as dashboards) in your application. For accessing the custom pages from your application, provide the name of the custom page as the value of the `perspective` parameter. Note that the `perspective` parameter is case-sensitive.
```
Comment (michelehaglund) [inline docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-standalone-custom-pages-proc.adoc]: I just looked up the usage of parentheses in the IBM Style Guide. It says to use them for abbreviations, symbols, and measurements, but to avoid them in running text. 

So, I would reword as follows:
Apart from standalone perspectives, you can also embed custom pages, also known as dashboards, in your application.

I'm no editor, so we may want to ask Emily of someone if that it too many commas :)

File: docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-standalone-custom-pages-proc.adoc [hunk 10]
```diff
@@ -0,0 +1,12 @@
...
+
+.Procedure
+. Login to {CENTRAL}.
```
Comment (michelehaglund) [inline docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-standalone-custom-pages-proc.adoc]: Log in to..

File: docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-standalone-custom-pages-proc.adoc [hunk 11]
```diff
@@ -0,0 +1,12 @@
...
+. In a web browser, enter the link of the custom page in the address bar, for example,
```
Comment (michelehaglund) [inline docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-standalone-custom-pages-proc.adoc]: instead of link, consider web address. Also, I looked up "URL" in the style guide and it says to avoid it.

Basically, it says "either use "web address" or do not use a term at all. For example, write "go to".. `http://localhost:8080/business-central/kie-wb.jsp?standalone=true&perspective=LibraryPerspective&header=UberfireBreadcrumbsContainer

File: docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-standalone-editor-proc.adoc [hunk 12]
```diff
@@ -0,0 +1,23 @@
+[id='using-standalone-perspectives-standalone-editor-proc']
+= Using standalone editor perspective
+
+You can use the standalone editor perspective of {CENTRAL} to access an asset's specific editor. By using this perspective, you can open an asset's editor and can modify the asset as required.
+
+The web link for accessing an asset's standalone editor perspective contains the `standalone` and `path` parameters. The `path` parameter must contain the complete path to the asset and then link must end with `_#StandaloneEditorPerspective_` string. Also, by varying the `path` parameter, you can access a specific asset's editor in the standalone mode.
```
Comment (michelehaglund) [inline docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-standalone-editor-proc.adoc]: "web link" should be "web address" as noted in comments above.

File: docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-standalone-editor-proc.adoc [hunk 13]
```diff
@@ -0,0 +1,23 @@
...
+
+.Procedure
+. Login to {CENTRAL}.
```
Comment (michelehaglund) [inline docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-standalone-editor-proc.adoc]: Log in to..

File: docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-standalone-editor-proc.adoc [hunk 14]
```diff
@@ -0,0 +1,23 @@
...
+. In a web browser, enter the appropriate link as required, for example,
```
Comment (michelehaglund) [inline docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-standalone-editor-proc.adoc]: "link" should be "web address" as noted in comments above.

File: docs/product-assembly_using-standalone-perspectives/src/main/asciidoc/main.adoc [hunk 15]
```diff
@@ -0,0 +1,30 @@
...
+= Using standalone perspectives
+// Context attribute is assembly specific and enables module reuse between assemblies.
+:imagesdir: topics/product-shared-docs/images
+:context: using-standalone-perspectives
+include::product-shared-docs/author-group.adoc[]
+
+// Purpose statement for the assembly
+As a developer, you can use the standalone perspectives from {CENTRAL} in your application.
+
+.Prerequisite
```
Comment (michelehaglund) [inline docs/product-assembly_using-standalone-perspectives/src/main/asciidoc/main.adoc]: Should be "Prerequisites" as there is more than one :)

File: docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-library-proc.adoc [hunk 16]
```diff
@@ -0,0 +1,20 @@
+[id='using-standalone-perspectives-library-proc']
+= Using standalone library perspective
```
Comment (michelehaglund) [inline docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-library-proc.adoc]: Using the..

File: docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-standalone-editor-proc.adoc [hunk 17]
```diff
@@ -0,0 +1,23 @@
+[id='using-standalone-perspectives-standalone-editor-proc']
+= Using standalone editor perspective
```
Comment (michelehaglund) [inline docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-standalone-editor-proc.adoc]: Using the..

File: docs/product-assembly_using-standalone-perspectives/src/main/asciidoc/main.adoc (hunk 15 above)
Comment (gadeyredhat) [reply docs/product-assembly_using-standalone-perspectives/src/main/asciidoc/main.adoc]: Changed as suggested.

File: docs/product-assembly_using-standalone-perspectives/src/main/asciidoc/main.adoc (hunk 1 above)
Comment (gadeyredhat) [reply docs/product-assembly_using-standalone-perspectives/src/main/asciidoc/main.adoc]: Changed as suggested.

File: docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-content-manager-proc.adoc (hunk 2 above)
Comment (gadeyredhat) [reply docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-content-manager-proc.adoc]: Changed as suggested.

File: docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-content-manager-proc.adoc (hunk 3 above)
Comment (gadeyredhat) [reply docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-content-manager-proc.adoc]: Changed as suggested.

File: docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-content-manager-proc.adoc (hunk 4 above)
Comment (gadeyredhat) [reply docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-content-manager-proc.adoc]: Changed as suggested.

File: docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-intro-con.adoc (hunk 5 above)
Comment (gadeyredhat) [reply docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-intro-con.adoc]: Changed as suggested.

File: docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-intro-con.adoc (hunk 6 above)
Comment (gadeyredhat) [reply docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-intro-con.adoc]: Changed as suggested.

File: docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-library-proc.adoc (hunk 16 above)
Comment (gadeyredhat) [reply docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-library-proc.adoc]: Changed as suggested.

File: docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-library-proc.adoc (hunk 7 above)
Comment (gadeyredhat) [reply docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-library-proc.adoc]: Changed as suggested.

File: docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-library-proc.adoc (hunk 8 above)
Comment (gadeyredhat) [reply docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-library-proc.adoc]: Changed partially. 
Michele, I have changed "web link" to "web address" but have not rephrased the sentences using "go to ..." for readability consistency.

File: docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-standalone-custom-pages-proc.adoc (hunk 9 above)
Comment (gadeyredhat) [reply docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-standalone-custom-pages-proc.adoc]: Changed as suggested.

File: docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-standalone-custom-pages-proc.adoc (hunk 10 above)
Comment (gadeyredhat) [reply docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-standalone-custom-pages-proc.adoc]: Changed as suggested.

File: docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-standalone-custom-pages-proc.adoc (hunk 11 above)
Comment (gadeyredhat) [reply docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-standalone-custom-pages-proc.adoc]: Changed partially.
Michele, I have changed "web link" to "web address" but have not rephrased the sentences using "go to ..." for readability consistency.

File: docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-standalone-editor-proc.adoc (hunk 17 above)
Comment (gadeyredhat) [reply docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-standalone-editor-proc.adoc]: Changed as suggested.

File: docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-standalone-editor-proc.adoc (hunk 12 above)
Comment (gadeyredhat) [reply docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-standalone-editor-proc.adoc]: Changed as suggested.

File: docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-standalone-editor-proc.adoc (hunk 14 above)
Comment (gadeyredhat) [reply docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-standalone-editor-proc.adoc]: Changed as suggested.
